airbyte-webapp/src/components/common/DataGeographyDropdown/DataGeographyDropdown.tsx

```diff
@@ -0,0 +1,60 @@
+import { faPlus } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { FormattedMessage, useIntl } from "react-intl";
+import { components, MenuListProps } from "react-select";
+
+import { DropDown } from "components/ui/DropDown";
+
+import { Geography } from "core/request/AirbyteClient";
+import { links } from "utils/links";
+
+import styles from "./DataGeographyDropdown.module.scss";
+
+interface DataGeographyDropdownProps {
+  geographies: Geography[];
+  isDisabled?: boolean;
+  onChange: (value: Geography) => void;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  value: any; // our current dropdown types options as any anyway
```

Comment:
Could we still type here as `Geography`? So we break the `any` polution with this component?

Reply:
Sure, I just updated the `value` and references to use `Geography`